### ISSUE TITLE
Standard booking type

### ISSUE THREAD

**Issue by bks0401** [meeting-room-booking-system/mrbs-code#3613]
Hi

Is it possible to change which type is default when booking.
Now it is "I", would like it was type "E" that was default

We use MRBS 1.9.4

/Bruno

**Describe the problem you're facing/question you have**
A clear and concise description of the problem you're facing, or the question you have.

**Browser details (please complete the following information):**

**Server details (please complete the following information):**
 - MRBS version [e.g. 1.9.4]
 - Web server: [e.g. Apache 2.4.48]
 - OS: [e.g. CentOS 7.9]
 - PHP version: [e.g. 7.4.23]
 - Database used: [e.g. MySQL 8.0.26]

**Additional context**

**Comment by campbell-m** [comment]
I can't remember if it's already there in MRBS 1.9.4, but try logging in to MRBS as an admin, going to the "Rooms" page and editing the area in question.  Then change the default type.  If not set

~~~php
$default_type = 'E';
~~~

in your config file.

**Comment by bks0401** [comment]
Hi

I have now update MRBS to version 1.11.4, and add this

$default_type = 'E';

in the config fil, but it is still "I" there is default

**Comment by campbell-m** [comment]
Yes, the config setting just gives the default setting for _new_ areas.  For _existing_ areas you need to login to MRBS as an admin, go to the Rooms page and edit the settings for the area.

**Comment by bks0401** [comment]
Thanks it works :-)
